main.go

```diff
@@ -36,12 +37,21 @@ var (
 	version string
 )
 
+//go:embed COPYING
+var copying string
+
 func main() {
 	flags := mgmt.Flags{
 		Debug:   Debug,
 		Verbose: Verbose,
 	}
-	if err := mgmt.CLI(program, version, flags); err != nil {
+	cliArgs := &mgmt.CLIArgs{
+		Program: program,
+		Version: version,
+		Copying: copying,
+		Flags: flags,
```

Comment:
We need #670 finished and merged. It would have picked this up!
```
main.go:52: File is not `goimports`-ed (goimports)
		Flags: flags,
```

```diff
diff --git a/main.go b/main.go
index 8eba6fc..8c36be1 100644
--- a/main.go
+++ b/main.go
@@ -49,7 +49,7 @@ func main() {
                Program: program,
                Version: version,
                Copying: copying,
-               Flags: flags,
+               Flags:   flags,
        }
        if err := mgmt.CLI(cliArgs); err != nil {
                fmt.Println(err)
```